tools/java/src/com/twitter/bazel/checkstyle/suppressions.xml

```diff
@@ -14,5 +14,6 @@
   -->
 
   <suppress files="Empty.java" checks=".*"/>
+  <suppress files="tests/.*" checks="JavadocMethod|JavadocType|JavadocStyle" />
```

Comment:
Can we keep `JavadocStyle` in place so if there are javadocs we verify they are valid.